<?xml version="1.0" ?>
<sdf version="1.6">
  <model name="large_gear">
    <static>false</static>
    <allow_auto_disable>false</allow_auto_disable>

    <link name="large_gear">
      <inertial auto="true"/>
      
      <visual name="large_gear_visual">
        <geometry>
          <mesh>
            <uri>model://demo_parts/large_gear/meshes/large_gear.stl</uri>
          </mesh>
        </geometry>
        <material>
          <ambient>1.0 1.0 1.0 1.0</ambient>
          <diffuse>1.0 1.0 1.0 1.0</diffuse>
        </material>
      </visual>

      <collision name="large_gear_collision">
        <density>10000</density>
        <pose>0 0 0 0 0 0</pose>
        <geometry>
          <mesh>
            <uri>model://demo_parts/large_gear/meshes/large_gear.stl</uri>
          </mesh>
        </geometry>
        <max_contacts>5</max_contacts>
        <surface>
          <contact>
            <ode>
              <max_vel>0.0</max_vel>
              <min_depth>0.1</min_depth>
            </ode>
          </contact>
          <friction>
            <ode>
              <mu>1000</mu>
              <mu2>1000</mu2>
            </ode>
          </friction>
        </surface>
      </collision>
    </link>
  </model>
</sdf>